### ISSUE TITLE
local-exec provisioner error

### ISSUE THREAD

**Issue by Mehdi-Leqsiouer** [terraform-google-modules/terraform-google-cloud-operations#105]
### TL;DR

Hello
I have this error when trying to run the module :
`
│ Error: local-exec provisioner error
│ 
│   with module.agent_policy.module.gcloud-destroy.null_resource.additional_components[0],
│   on .terraform/modules/agent_policy.gcloud-destroy/main.tf line 175, in resource "null_resource" "additional_components":
│  175:   provisioner "local-exec" {
│ 
│ Error running command
│ '.terraform/modules/agent_policy.gcloud-destroy/scripts/check_components.sh
│ gcloud beta': exit status 127. Output: 
╵
`
`
╷
│ Error: local-exec provisioner error
│ 
│   with module.agent_policy.module.gcloud-upsert.null_resource.additional_components[0],
│   on .terraform/modules/agent_policy.gcloud-upsert/main.tf line 175, in resource "null_resource" "additional_components":
│  175:   provisioner "local-exec" {
│ 
│ Error running command
│ '.terraform/modules/agent_policy.gcloud-upsert/scripts/check_components.sh
│ gcloud beta': exit status 127. Output: 
╵
`

![image](https://github.com/terraform-google-modules/terraform-google-cloud-operations/assets/46991431/f557e161-450a-4ae9-9386-adf5a000fbb8)

### Terraform Configuration

```hcl
module "agent_policy" {
  source  = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version = "~> 0.4.0"

  project_id = var.project_id
  zones      = [var.zone]
  policy_id  = "ops-agents-policy"
  agent_rules = [
    {
      type               = "ops-agent"
      version            = "current-major"
      package_state      = "installed"
      enable_autoupgrade = true
    },
  ]
  os_types = [
    {
      short_name = "debian"
      version    = "11"
    },
  ]
}
```

### Terraform Version

```sh
terraform {
  required_version = ">= 1.3.5"
}
```
